xml/Makefile.am

```diff
@@ -162,6 +162,21 @@ api/api-result-%.rng: $(API_build_copies) best-match.sh Makefile.am
 	$(AM_V_at)echo '  </start>' >> $@
 	$(AM_V_SCHEMA)echo '</grammar>' >> $@
 
+crm_mon.rng: api/crm_mon-$(MON_max).rng
+	$(AM_V_at)echo '<?xml version="1.0" encoding="UTF-8"?>' > $@
+	$(AM_V_at)echo '<grammar xmlns="http://relaxng.org/ns/structure/1.0"' >> $@
+	$(AM_V_at)echo '         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">' >> $@
+	$(AM_V_at)echo '    <start>' >> $@
+	$(AM_V_at)echo '        <ref name="element-crm_mon-old"/>' >> $@
+	$(AM_V_at)echo '    </start>' >> $@
+	$(AM_V_at)echo '    <define name="element-crm_mon-old">' >> $@
+	$(AM_V_at)echo '        <element name="crm_mon">' >> $@
+	$(AM_V_at)echo '            <attribute name="version"> <text/> </attribute>' >> $@
+	$(AM_V_at)echo '            <externalRef href="$(<)" />' >> $@
+	$(AM_V_at)echo '        </element>' >> $@
+	$(AM_V_at)echo '    </define>' >> $@
+	$(AM_V_at)echo '</grammar>' >> $@
```

Comment:
The final line should use AM_V_SCHEMA. This is a hack for files created by multiple lines.

This comes into play if someone does "./configure --enable-quiet" or "make V=0". Normally, make echoes every line it executes, but with one of those options, all of the AM_V_at lines will be silent and AM_V_SCHEMA will output "SCHEMA *filename*" to indicate that the file was generated.

Reply:
Actually maybe it's ./configure --enable-silent-rules. The abundance of "quiet" options is confusing ...

Reply:
I got into a bit of a rhythm ther with the copy & paste at the front of each line.  Fixed and verified with "make V=0".